create-ami.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/ksh
+#git@github.com:shazaum/aws-openbsd.git!/bin/ksh
```

Comment:
that shouldn't be part of the changes ;-)

Reply:
lol... that doesn't make sense...
I don't know how it got there.
I'll fix

Reply:
fixed